### PR TITLE
Branch add view command to ug

### PR DESCRIPTION
**Pull request by Nicholas-Cheng-De-Fei**
Update use case, since the user does not need to view all users since the users are already shown on screen,

image

This is how the `view` command function looks like in the user guide,

image

Closes https://github.com/AY2425S1-CS2103-F10-3/tp/issues/149

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -153,6 +153,20 @@ Clears all entries from the address book.
 
 Format: `clear`
 
+### Clearing all entries : `view`
```
Comment (sheenkerr) [inline docs/UserGuide.md:156]: I think the header here should be something like "View an existing contact's details"?

**Review by sheenkerr — CHANGES_REQUESTED** [review]

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -364,17 +364,16 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
 
     Use case ends.
 
 **Extensions**
 
-- 2a. The specified person is invalid.
+- 1a. The specified person is invalid.
 
-  - 2a1. System shows an error message.
+  - 1a1. System shows an error message.
```
Comment (ITLimJiaWei) [inline docs/DeveloperGuide.md:376]: Should it not be 2a? If I recall correctly, it was discussed during a tutorial

**Review by ITLimJiaWei — CHANGES_REQUESTED** [review]
Change as requested above

**Review by ITLimJiaWei — APPROVED** [review]
LGTM

**Review by sheenkerr — APPROVED** [review]
LGTM

**Pull request merged**